src/main/java/io/github/jhipster/online/mdc/logging/MDCInterceptor.java

```diff
@@ -0,0 +1,23 @@
+package io.github.jhipster.online.mdc.logging;
+
+import io.github.jhipster.online.security.SecurityUtils;
+import org.slf4j.MDC;
+import org.springframework.web.servlet.AsyncHandlerInterceptor;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class MDCInterceptor implements AsyncHandlerInterceptor {
+    @Override
+    public boolean preHandle(HttpServletRequest request, HttpServletResponse response, Object handler) {
+        if (SecurityUtils.getCurrentUserLogin().isPresent())
+            MDC.put("user.login", SecurityUtils.getCurrentUserLogin().get());
```

Comment:
"user.login" could be a constant.

Personally, I would have not used a 'dot' notation, only a simple user or userId.

Reply:
Thanks. Changed to `user`. :+1: